### DISCUSSION TITLE
Mortal_vs_Mortal 硬件利用率不足

### DISCUSSION THREAD

**Discussion by hyskylord** [Equim-chan/Mortal#80] [Q&A]
最近想进行大规模的对战来比较模型性能，所以特别关注了一下运行1V3的时候的硬件利用率，数据如下：
CPU利用率：约25%
GPU利用率：约30%
对战时间：500场对局/分钟

一开始我以为是并行数不够，所以增加了同时进行的对战局数，但是利用率和速度只增加了一点点。这似乎很奇怪，因为自我对战应该是多线程利用率非常高的。

EDIT:上传了一张Windows系统下的CPU利用率图，另一台Linux系统情况差不多。
![1 XH@(` 8EVHV%42`565WIJ](https://github.com/user-attachments/assets/6a618122-1dad-4450-8135-a92681480774)

**Comment by Equim-chan** [comment]
games_per_iter 有多少？

**Reply by hyskylord** [reply]
测试了2000到10000

**Reply by Equim-chan** [reply]
可以试试手动开多个进程。

**Reply by hyskylord** [reply]
手动多开确实解决了问题。
